fix: run `reroute` in a middleware function before split functions (#16590)
sveltejs/kit: documentation/docs/25-build-and-deploy/80-adapter-netlify.md

	directory = "functions"
```

## Notes

### Individual functions and `reroute`

> If the `split` option is set to `true` for a route, or at the adapter level, the [`reroute`](hooks#reroute) function will be deployed as an edge middleware that runs before any individual function.

Author: ```suggestion
If the adapter `split` option is set to `true`, the [`reroute`](hooks#reroute) hook will be deployed as an edge middleware that runs before any individual function.
```
